start.js

```diff
@@ -35,14 +35,16 @@ async function collectDataUsage () {
 
 function createSessionDataDefaults () {
 // Create template session data defaults file if it doesn't exist
-  const dataDirectory = path.join(__dirname, '/app/data')
+  const dataDirectory = path.join('.', '/app/data')
   const sessionDataDefaultsFile = path.join(dataDirectory, '/session-data-defaults.js')
   const sessionDataDefaultsFileExists = fs.existsSync(sessionDataDefaultsFile)
 
   if (!sessionDataDefaultsFileExists) {
     console.log('Creating session data defaults file')
     if (!fs.existsSync(dataDirectory)) {
-      fs.mkdirSync(dataDirectory)
+      fs.mkdirSync(dataDirectory, {
+        recursive: true
```

Comment:
Should we remove the check for the directory existing as `recursive` makes it safe to attempt the creation of a directory which already exists?

I'm assuming `recursive` has been added for any cases where `app` directory has not yet been created, might this lead to difficulty later when the user tries to copy the boilerplate `app` directory and a mostly empty on already exists?

Reply:
Done as requested.